frontends/android/BitBoxApp/app/src/main/java/ch/shiftcrypto/bitboxapp/GoViewModel.java

```diff
@@ -170,14 +169,7 @@ public void bluetoothConnect(String identifier) {
 
         @Override
         public boolean usingMobileData() {
-            // Adapted from https://stackoverflow.com/a/53243938
```

Comment:
Also it seems you dropped this comment. I prefer to keep comments unless they don't apply anymore :smile: